proto-lens-protoc/package.yaml

```diff
@@ -55,6 +55,7 @@ library:
     - Data.Default.Class as Data.ProtoLens.Reexport.Data.Default.Class,
     - Data.Map as Data.ProtoLens.Reexport.Data.Map,
     - Data.ProtoLens as Data.ProtoLens.Reexport.Data.ProtoLens,
+    - Data.ProtoLens.GRPC.Types as Data.ProtoLens.Reexport.Data.ProtoLens.GRPC.Types,
```

Comment:
Let's call this module `Data.ProtoLens.Service.Types`, since there are multiple users of the "service" spec (in particular Google's internal Stubby protocol).